scripts/quality/run_qlty_zero.py

```
@@ -138,12 +138,11 @@ def cast_mapping(value: Any) -> Mapping[str, Any]:
 def _run_qlty_check(repo_dir: Path) -> subprocess.CompletedProcess[str]:
     """Handle run qlty check."""
     executable_path = _resolved_qlty_executable_path()
-    command = _build_qlty_check_argv()
     # Safe-by-construction: a fixed literal executable name, explicit argv,
     # shell=False, and an absolute executable path supplied separately.
+    # nosemgrep: python.lang.security.audit.dangerous-subprocess-use-audit.dangerous-subprocess-use-audit
     return subprocess.run(  # nosec B603
-        # nosemgrep
-        command,
+        ["qlty", "check", "--all", "--fail-level", "note", "--summary"],
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)